sql-statements/sql-statement-calibrate-resource.md

```diff
@@ -102,6 +106,20 @@ CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
 ERROR 1105 (HY000): The workload in selected time window is too low, with which TiDB is unable to reach a capacity estimation; please select another time window with higher workload, or calibrate resource by hardware instead
 ```
 
+The capacity estimation needs to Prometheus metric data, including `resource_manager_resource_unit`, `process_cpu_usage`, `tikv_cpu_quota`, and `tidb_server_maxprocs`. If the corresponding metrics data is empty, an error is displayed indicating the name of the metrics item as shown in the following example. And `resource_manager_resource_unit` is empty when no workload, so this error will also be reported.
+
+```sql
+CALIBRATE RESOURCE START_TIME '2023-04-18 08:00:00' DURATION '60m';
+Error 1105 (HY000): metrics 'resource_manager_resource_unit' is empty
+```
+
+TiKV does not monitor CPU usage  metrics on macOS. An error indicating the name of the metrics item is shown in the following example.
```

Comment:
```suggestion
Because TiKV does not monitor CPU usage on macOS, it reports an error that the corresponding monitoring data is empty, as shown in the following example.
```